apko.yaml

```diff
@@ -0,0 +1,52 @@
+contents:
+  repositories:
+    - https://packages.wolfi.dev/os
```

Comment:
nit: you don't need this if you pass it in `--repository-append`

Reply:
Oh good catch, will remove

Reply:
So I flipped this around and added the key and repo to `apko.yaml` - this way building directly with `apko` also works:
```
apko build --arch host apko.yaml academy.local academy.tar
```